controllers/upgrade.go

```diff
@@ -97,6 +108,11 @@ func (r *RollingUpgradeReconciler) ReplaceNodeBatch(rollingUpgrade *v1alpha1.Rol
 			// Is drained?
 
 			// Terminate - set lastTerminateTime
+			if err := r.Auth.TerminateInstance(target); err != nil {
+				r.Info("Instance termination failed. Will retry in next reconcile", "name", rollingUpgrade.NamespacedName(), "instance", aws.StringValue(target.InstanceId))
+				return true, nil
+			}
+			r.SetInstanceTag(rollingUpgrade, target, instanceStateTagKey, completedTagValue)
```

Comment:
Yeah, so we can probably not do this once we successfully terminate, we don't care about the instance any more, and it would be a waste of API call

Reply:
I agree. Removing this.